src/components/pages/doc.less

```diff
@@ -1,11 +1,19 @@
+// Import the @control-panel-height variable
+@import '../control-panel.less';
+
+// `has:` works in all modern browsers except Firefox
+*:has(> .DocPage-controlPanels) > *:first-child {
+  min-height: %(~"calc(100vh - %s)", @control-panel-height);
+}
+
```

Comment:
Some cosmetics that remove the scrollbar on pages with little content (except for Firefox)